llarp/dht/localrouterlookup.cpp

```diff
@@ -43,7 +43,11 @@ namespace llarp
             found = rc;
         }
         valuesFound.clear();
-        valuesFound.emplace_back(found);
+        if(not found.pubkey.IsZero())
+        {
+          valuesFound.resize(1);
+          valuesFound[0] = found;
+        }
       }
```

Comment:
Maybe a log statement here? This isn't a case we should be hitting, so it deserves attention when we do.

Reply:
Added in #1041